webviz_config/_plugin_abc.py

```diff
@@ -332,14 +325,18 @@ def get_all_settings(self) -> List[html.Div]:
     def _plugin_wrapper_id(self) -> str:
         return f"plugin-wrapper-{self._plugin_unique_id}"
 
+    @property
+    def _legacy_plugin_view_id(self) -> str:
+        return f"{self._plugin_unique_id.to_string()}-view"
```

Comment:
The `@property` (indicating a public property) and the leading`_` (indicating private attribute) is a bit conflicting perhaps?

Reply:
Converted the property to a private attribute in `__init__`